controllers/constant/odlm.go

```diff
@@ -855,6 +858,11 @@ spec:
             walStorage:
               size: 1Gi
         force: true
+        annotations:
+          k8s.enterprisedb.io/addons: ["velero"]
+          k8s.enterprisedb.io/snapshotAllowColdBackupOnPrimary: enabled
+        labels:
+          foundationservices.cloudpak.ibm.com: keycloak
```

Comment:
less important but I think this should be keycloak-data to match what is already present on the scripts side

Reply:
that's fine, we can have two separate labels to differentiate between pgdump approach and velero addon approach since they shouldn't conflict. It is easier to change on the scripts side anyways so once we change to fixed EDB version we can make the change on the scripts side.